packages/documentation/src/stories/misc/migration-guide/migrationv4-5.component.ts

```diff
@@ -159,14 +159,10 @@ export class MigrationV45Component extends LitElement {
   private _templateBootstrapInstructions() {
     return html`
       Update Bootstrap to version
-      <a href="https://getbootstrap.com/docs/5.1/migration">5.1.x</a>
+      <a href="https://getbootstrap.com/docs/5.1/migration">Bootstrap 5.1.x Migration Guide</a>
```

Comment:
The sentence is now "Update Bootstrap to version Bootstrap 5.1.x Migration Guide and ng-bootstrap to version ng-bootstrap 12.x.x Changelog".
Can you update it in all migration sections so that it makes sense?